src/robot/output/listeners.py

```diff
@@ -64,14 +64,14 @@ def __nonzero__(self):
 
     def _import_listeners(self, listener_data):
         listeners = []
-        for name, args in listener_data:
+        for listener, args in listener_data:
             try:
-                listeners.append(ListenerProxy(name, args))
+                listeners.append(ListenerProxy(listener, args))
             except DataError as err:
                 if args:
-                    name += ':' + ':'.join(args)
+                    listener += ':' + ':'.join(args)
```

Comment:
This breaks if `listener` is isn't a s string. I'm not sure is it possible that creating `ListenerProxy` could actually fail in that case, though.